library/alloc/src/alloc.rs

```diff
@@ -36,7 +36,7 @@ extern "Rust" {
 /// if there is one, or the `std` crate’s default.
 ///
 /// Note: while this type is unstable, the functionality it provides can be
-/// accessed through the [free functions in `alloc`](index.html#functions).
+/// accessed through the [free functions in `alloc`](super#functions).
```

Comment:
Hmm, I think this is actually meant to go to `alloc::alloc`, not `alloc` the root crate: https://doc.rust-lang.org/nightly/alloc/alloc/
```suggestion
/// accessed through the [free functions in `alloc`](self#functions).
```

Reply:
You marked this as resolved but forgot to change it ;)
```
 std/alloc/struct.Global.html:9: broken link fragment `#functions` pointing to `alloc/index.html`
alloc/alloc/struct.Global.html:9: broken link fragment `#functions` pointing to `alloc/index.html`
```

Reply:
But I did! That's what's weird, this is the _current_ code:

```rust
/// accessed through the [free functions in `alloc`](self#functions).
```

Reply:
Could this be a bug in rustdoc?

Reply:
I made this comment before you pushed the code. Try to avoid marking comments as resolved before you push the commits addressing them.

Reply:
Hmm, I thought I marked them as resolved when I pushed 🤷 . I'll try to in the future :)

Reply:
It's always possible I was looking at an old commit 😅